### PR TITLE
added custom filters

### PR DESCRIPTION
**Pull request by lucawperri**

### CODE REVIEW

**Comment by PaytonOAnderson** [comment]
Looks good to me.

File: packages/react-frontend/src/taskList/Filters.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+function Filters(props) {
+    const uniqueTags = Array.from(
+      new Set(
+        (props.tasks || [])
```
Comment (Proefey) [inline packages/react-frontend/src/taskList/Filters.js:4]: This part of the code is a bit difficult to understand, maybe add a comment to increase clarity

Comment (Proefey) [reply packages/react-frontend/src/taskList/Filters.js:4]: Otherwise Good Job On The Code!

**Comment by NipunDas** [comment]
The code and the deployed version looks good to me. @lucawperri Have you tested the filtering with multiple tags for a task yet, or is each task meant to only have one tag?

**Comment by Muhammad8493** [comment]
The code looks good and works as expected.

**Review by Muhammad8493 — APPROVED** [review]

**Comment by lucawperri** [comment]
> The code and the deployed version looks good to me. @lucawperri Have you tested the filtering with multiple tags for a task yet, or is each task meant to only have one tag?

It supports multiple tags for tasks, but I don't think that's been implemented on the addtask page yet.

**Pull request merged**